source/scripts/model/Monster.js

```diff
@@ -100,6 +100,49 @@ export default class Monster {
         this.position.x += movement.x
         this.position.y += movement.y
     }
+    pursuit(magnitude) {
+
+          //Variables for the monster and hero positions
+       var monX = this.position.x
+       var monY = this.position.y
+
+       var adX = this.game.adventurer.position.x 
+       var adY = this.game.adventurer.position.y
+
+       var dx = this.game.adventurer.position.x - this.position.x
+       var dy = this.game.adventurer.position.y - this.position.y
+
+       
+       var magnitude
```

Comment:
Aah, I see. But does it need to be re-initialized here? If it is already passed in as an argument?